ci/requirements/min-all-deps.yml

```diff
@@ -8,38 +8,38 @@ dependencies:
   # When upgrading python, numpy, or pandas, must also change
   # doc/user-guide/installing.rst, doc/user-guide/plotting.rst and setup.py.
   - python=3.8
-  - boto3=1.18
+  - boto3=1.20
   - bottleneck=1.3
-  - cartopy=0.19
+  - cartopy=0.20
   - cdms2=3.1
   - cfgrib=0.9
   - cftime=1.5
   - coveralls
-  - dask-core=2021.09
-  - distributed=2021.09
+  - dask-core=2021.11
+  - distributed=2021.11
   - flox=0.5
   - h5netcdf=0.11
   # h5py and hdf5 tend to cause conflicrs
   # for e.g. hdf5 1.12 conflicts with h5py=3.1
   # prioritize bumping other packages instead
-  - h5py=3.1
-  - hdf5=1.10
+  - h5py=3.6
+  - hdf5=1.12
   - hypothesis
   - iris=3.1
   - lxml=4.6  # Optional dep of pydap
-  - matplotlib-base=3.4
-  - nc-time-axis=1.3
+  - matplotlib-base=3.5
+  - nc-time-axis=1.4
   # netcdf follows a 1.major.minor[.patch] convention
   # (see https://github.com/Unidata/netcdf4-python/issues/1090)
   # bumping the netCDF4 version is currently blocked by #4491
-  - netcdf4=1.5.3
+  - netcdf4=1.5
```

Comment:
I think this is still blocked by the comment above?

Reply:
You are correct. I was experimenting with ways to silence this warning: 

```
Analyzing netcdf4...
WARNING: patch version should not appear in requirements file: netcdf4
```

Reply:
I looked into this a bit more. This comment was misleading. Now that we have pulled pynio from this test environment, we can bump the netcdf4 version. (1.5.3 was released on Oct 22, 2019).